FMI2QGIS/ui/wms_dialog.py

```diff
@@ -70,6 +75,25 @@ def __refresh_wms_layers(self):
             abstract_item.setToolTip(wms_layer.abstract)
             self.tbl_wms_layers.setItem(idx, 2, abstract_item)
 
+    def __search_wms_layers(self):
+        self.wms_layers = self.wms_layer_handler.list_wms_layers()
+        self.tbl_wms_layers.setColumnCount(3)
+        self.tbl_wms_layers.setRowCount(len(self.wms_layers))
+        self.search_string = self.ln_ed_wms_search.value()
+
+        for idx, wms_layer in enumerate(self.wms_layers):
+            string_to_match = wms_layer.name.lower()
```

Comment:
Same here, could title and abstract be used as well?

Reply:
The same here. It could be useful for some layers.

Reply:
I noticed that for example `kesälahti` wasn't working since `ä` was not included in the name but only in the title